clang/include/clang/Analysis/FlowSensitive/Formula.h

```diff
@@ -87,6 +87,9 @@ class alignas(const Formula *) Formula {
                          ArrayRef<const Formula *> Operands,
                          unsigned Value = 0);
 
+  // Parse Formulas using Arena rather than caling this function directly.
```

Comment:
nit: Maybe prefix with "note:" or somesuch? I started reading this as a description of the function.